lightwood/mixer/neural.py

```diff
@@ -302,14 +305,15 @@ def partial_fit(self, train_data: EncodedDs, dev_data: EncodedDs) -> None:
         """
 
         # Based this on how long the initial training loop took, at a low learning rate as to not mock anything up tooo badly # noqa
+        self.started = time.time()
         train_dl = DataLoader(train_data, batch_size=self.batch_size, shuffle=True)
         dev_dl = DataLoader(dev_data, batch_size=self.batch_size, shuffle=True)
         optimizer = self._select_optimizer()
         criterion = self._select_criterion()
         scaler = GradScaler()
 
         self.model, _, _ = self._max_fit(train_dl, dev_dl, criterion, optimizer, scaler,
-                                         self.stop_after, max(1, int(self.epochs_to_best / 3)))
+                                         self.stop_after * 0.1, max(1, int(self.epochs_to_best / 3)))
```

Comment:
To confirm: this implies final tuning uses 10% of the budget?

Reply:
Yes